rogue_net/rogue_net/head_creator.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict
+from torch import nn
+
+from entity_gym.environment import ActionSpace, CategoricalActionSpace
+
+
+def create_head_for(space: ActionSpace) -> nn.Module:
```

Comment:
This should probably take `d_model` as an input?

Reply:
This has been changed